lib/internal/inspector_async_hook.js

```diff
@@ -0,0 +1,63 @@
+'use strict';
+
+const { createHook } = require('async_hooks');
+const inspector = process.binding('inspector');
+
+if (!inspector || !inspector.asyncTaskScheduled) {
```

Comment:
if the inspector is disabled at compile time, binding will return falsy, not throw?